react/features/base/oauth/functions.js

```diff
@@ -0,0 +1,29 @@
+// @flow
+
+import { getJitsiMeetGlobalNS } from '../util';
+
+
+/**
+ * Executes the oauth flow.
+ *
+ * @param {string} authUrl - The URL to oauth service.
+ * @returns {Promise<string>} - The URL with the authorization details.
+ */
+export function authorize(authUrl: string): Promise<string> {
+    const windowName = `oauth${Date.now()}`;
```

Comment:
Is there anything that prevents multiple auth windows for the same url from being opened? Does anything handle the user closing the dialog? For outlook, I had to add an interval to check if the window was closed while on the auth flow. I also stored the window so I could focus on it if the user tried to click the auth window again. This is one place where the divergence of implementations, which has I think been agreed on to be resolved after both features are merged, may be tricky.

Reply:
Those use cases are not handled. I'm wondering if I need to handle them. 

On close it would be good to clean up the callback. But should we prevent the user for opening multiple windows?

PS: I'm not a big fan of polling. IMHO if this would be the cost to implement it, i'd rather leave those use cases unhandled.